CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## How to contribute
+So you want to write code and get it landed in the official Firefox UI Tests repository? Then first fork [our repository](https://github.com/mozilla/firefox-ui-tests) into your own Github account, and create a local clone of it as described in the [installation instructions](https://github.com/mozilla/firefox-ui-tests#installation). The latter will be used to get new features implemented or bugs fixed. Once done and you have the code locally on the disk, you can get started. It's best not to work directly on the master branch, but to create a separate branch for each issue you are working on. That way you can easily switch between different work, and you can update each branch individually with the latest changes on upstream master . Check also our [best practices for Git](http://ateam-bootcamp.readthedocs.org/en/latest/reference/git_github.html).
+
+### Writing Code
+Please follow our [Python style guide](http://ateam-bootcamp.readthedocs.org/en/latest/reference/python-style.html), and also test with [pylama](https://pypi.python.org/pypi/pylama). If something is unclear, look at existing code which might help you to understand it better.
+
+### Submitting Patches
+When you think the code is ready for review, a pull request should be created on Github. Owners of the repository will watch out for new PR's and review them at regular intervals. For each update to the PR, we automatically run all the tests via [Travis CI](http://travis-ci.org/). If tests are failing, it's best to address the failures before requesting review. Otherwise you can wait for a review. If comments have been given in a review, they have to get integrated. For those changes, a separate commit should be created and pushed to your remote development branch. Don't forget to add a comment in the PR afterward, so everyone gets notified by Github. Keep in mind that reviews can span multiple cycles until the owners are happy with the new code.
```

Comment:
So we have PRs on Github but reviews are done via Bugzilla. So we should add a sentence how to create a text attachment and set the r? flag appropriately.